src/tree_cmd.cpp

```diff
@@ -805,6 +859,110 @@ CommandCost CmdPlantTree(DoCommandFlags flags, TileIndex end_tile, TileIndex sta
 	}
 }
 
+
+int _tree_placer_cmd_count = 0;
+/**
+ * Sync trees, sent when a client is using the Tree Placer.
+ * @param flags type of operation
+ * @param cmd_data stuff
+ * @return the cost of this operation or an error
+ */
+CommandCost CmdSyncTrees(DoCommandFlags flags, const TreeSyncCmdData &cmd_data)
+{
+	if (_networking && cmd_data.calling_client == _network_own_client_id && flags.Test(DoCommandFlag::Execute)) {
+		_tree_placer_cmd_count--;
```

Comment:
This looks like it will result in it going negative, as ResetToolData clears it to 0, and then the commands from the server will presumably be arriving later.